store.js

```diff
@@ -0,0 +1,101 @@
+// this file with store my questions and answers
+
+const STORE = [
+  {
+    question: 'Who currently holds the spot for the greatest golfer of all time, based on total tournament victories?',
+    answers: [
+      'Jack Nicklaus',
+      'Tiger Woods',
+      'Arnold Palmer',
+      'Sam Snead'
+    ],
+    correctAnswer: 'Sam Snead'
```

Comment:
Whole thing looks good, nice and clean. One issue I see, though, is in future-proofing this. (Whenever I see something complicated repeated AND necessary to functionality, it's a small code smell.)

What if a later developer comes in here and adds a new question, or edits an existing question, and accidentally types the correct `answer` and `correctAnswer` differently? It'd be easy to miss that in a PR but it'd ultimately break the whole quiz. I'd recommend obfuscating your strings from your `correctAnswer` functionality.

Couple of ideas come to mind. First:
```js
{
  question: "Who...",
  answers: [
    "Jack Nicklaus",
    "Tiger Woods",
    "Arnold Palmer",
    "Sam Snead"
  ],
  correctAnswer: () => this.answers[3]
}
```
But, then if the order ever changes you've always broken the functionality. Plus, it'd mean `correctAnswer` is a function that needs invoking.

Another thought is to have a list of wrong answers and one right one:
```js
{
   wrongAnswers: [...]
   correctAnswer: "Sam Snead"
}
```
Then when you want to iterate over all the answer you'd do something like `[...wrongAnswer, correctAnswer]`. Main  issue is just that if you use a for loop you'd end up with the right answer ALWAYS at the end of the array. That could be solved by randomizing the order afterward, or by using .forEach, which _should_ sufficiently randomize them.

(FYI, the splat operator with wrong answer (`...wrongAnswers`) just means "grab all the items in the `wrongAnswers` array and put them in this array."

I think the best case scenario would simply be to have a list of objects. (A point that could be debated but I'd probably go down this route.)
```js
{
  question: "Who...",
  answers: [
    { value: "Jack Nicklaus", isCorrect: false },
    { value: "Tiger Woods", isCorrect: false },
    { value: "Arnold Palmer", isCorrect: false },
    { value: "Sam Snead", isCorrect: true },
  ]
}
```
Then you could easily iterate over them and when checking the answer you'd just check if the matching `answer.isCorrect` is true.

Reply:
I see what you're saying, and I like it! Your final recommendation will also help me write the JavaScript because I am more familiar with this layout. I will make the necessary changes.

Reply:
Updated the code!